### PR TITLE
glog: new package

### PR DESCRIPTION
**Pull request by cpanato**
- glog: new package

Related: https://github.com/chainguard-dev/image-requests/issues/359

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

#### For new version streams
- [x] The package (and subpackages) `provides:` logical unversioned forms of the package (e.g. `nodejs`, `nodejs-lts`)

### CODE REVIEW

File: glog.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+  - uses: cmake/build
+
+  - uses: cmake/install
+
+  - uses: strip
+
+subpackages:
+  - name: "glog-dev"
+    description: "multi-language toolbox for accelerated data interchange and in-memory processing (development files)"
```
Comment (dlorenc) [inline glog.yaml:47]: Is this right?

File: glog.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    pipeline:
+      - uses: split/dev
+    dependencies:
+      provides:
+        - glog-dev=${{package.full-version}}
```
Comment (rawlingsj) [inline glog.yaml:52]:
```suggestion
    dependencies:
      runtime:
        - glog
```
melange will auto add the provides

File: glog.yaml (hunk 1 above)
Comment (cpanato) [reply glog.yaml]: no, copy and paste issue

fixed

File: glog.yaml (hunk 2 above)
Comment (cpanato) [reply glog.yaml]: fixed

File: glog.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+      - uses: split/dev
+    dependencies:
+      runtime:
+        - glog
+
+update:
+  enabled: true
+  github:
+    identifier: google/glog
+    use-tag: true
```
Comment (stormqueen1990) [inline glog.yaml:58]: It seems this project uses releases -- should `use-tag` be omitted so the releases are pulled instead?

Comment (cpanato) [reply glog.yaml:58]: ahh, i see, i will drop that

Comment (cpanato) [reply glog.yaml]: done

**Review by stormqueen1990 — APPROVED** [review]

**Pull request merged**